src/Illuminate/View/composer.json

```diff
@@ -15,7 +15,7 @@
     ],
     "require": {
         "php": "^8.0.2",
-        "ext-json": "*",
+        "ext-tokenizer": "*",
```

Comment:
See usages of `token_get_all`, `T_CLOSE_TAG`, `T_INLINE_HTML`, `T_OPEN_TAG`, and `T_OPEN_TAG_WITH_ECHO`.